pom.xml

```diff
@@ -11,8 +11,8 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
 
-    <beam.version>2.32.0</beam.version>
-    <connector.version>2.33.0.1-SNAPSHOT</connector.version>
+    <beam.version>2.35.0</beam.version>
+    <connector.version>2.35.0</connector.version>
```

Comment:
The Connector is not released yet, so we need to leave it as `2.33.0.1-SNAPSHOT`